index.html

```diff
@@ -169,19 +169,20 @@
 The <dfn id="netinfo.type">`type`</dfn> attribute, when getting, returns the <a title="connection types">connection type</a> that the user agent is using.
 
 ### `downlinkMax` attribute
-The <dfn>`downlinkMax`</dfn> attribute represents the <a>maximum downlink speed</a>, in <a>megabits per second</a> (Mbps), for <a>underlying connection technology</a>.
+The <dfn>`downlinkMax`</dfn> attribute represents the <a>maximum downlink speed</a>, in <a>megabits per second</a> (Mbps), of the first network hop, which is determined by the properties of the <a>underlying connection technology</a>.
 
-</section>
-
-<section>
+<div class="note">
+The user agent has no knowledge of the total number or the performance characteristics of the various network hops required to fulfill a particular request; different requests may follow different route and have different performance characteristics. The reported <dfn>maximum downlink speed</dfn> value is the maximum data rate of the first network hop, as determined by the properties of the <a>underlying connection technology</a>. The end-to-end performance of any request cannot exceed this value, but it is also not a guarantee of performance.
```

Comment:
"is the maximum data rate of the first network hop" -> "is an upper bound on the maximum data rate of the first network hop". Otherwise the value is a lie.